scene/gui/gradient_edit.cpp

```diff
@@ -297,6 +288,12 @@ void GradientEdit::_notification(int p_what) {
 			picker->connect("color_changed", callable_mp(this, &GradientEdit::_color_changed));
 		}
 	}
+
+	if (p_what == NOTIFICATION_ENTER_TREE || p_what == NOTIFICATION_THEME_CHANGED) {
+		draw_spacing = 3 * get_theme_default_base_scale();
+		draw_point_width = 8 * get_theme_default_base_scale();
```

Comment:
Maybe should've kept the default values as consts or defines?

Reply:
Yeah having them as constants in the header would make sense IMO.